[ADD] Barcode: Add scanning RFID tags documentation (#15350)
odoo/documentation: content/applications/inventory_and_mrp/barcode/operations/scan_rfid.rst

==========================================
Scanning RFID barcodes to manage inventory
==========================================

Make sure the RFD40 scanner is paired and connected to the mobile computer or smartphone. Scanning
the barcode pulls up the relevant inventory or manufacturing documents in Odoo on the mobile device.

Overview of the workflow
========================

#. With the RFD40 scanner in hand, open the **Barcode** app on the computer or phone.
#. Open the operation record (via the **Barcode** interface or scanning the operation barcode).
#. Scan the RFID barcodes for the product.

Manage inventory
----------------

To begin managing inventory using RFID tags, open the **Barcode** app on the smart device.

From there, scan the operation barcode or tap the :guilabel:`Operations` button to open operations
for receipts, internal transfers, delivery orders, or manufacturing orders. If the reading device is
not connected, Odoo will connect to the device. When the device is connected, a toast notification

> appears on the screen that verifies the connection. Scan the products, then validate the scan on the

Reviewer: ```suggestion
appears on the screen that verifies the connection. Scan the desired products, then validate the scan on the
```

Author: We're not really scanning products individually; it's more a matter of the reader picking up on the radio waves that happen to be nearby. So it'll grab all products, not just one at a time, if that makes sense. For that reason, I don't want to use the word "desired" here -- there's no "just these are the ones I want" associated with it.

mobile computer or smartphone.

.. image:: scan_rfid/validate-receipt.png
   :alt: Validate the operation on the mobile device.

Counting inventory
------------------

> Count inventory in the **Barcode** app by opening the :guilabel:`Inventory count`. Scan the products

Reviewer: ```suggestion
It is required by many factories and warehouses to keep track of inventory, including regular counts of products to ensure all records are correct. To change the amount of a product in stock, or *adjust inventory*, open the **Barcode** app and tap or click the :guilabel:`Inventory count` operation.

Scan the product barcode, then type the new number in the field. When updated, click or tap :guilabel: `Apply`.
```

I am only guessing at the steps, but I find it to be best practice to include all the steps for a section in case a user jumps to that part, just to learn how to do 'X'. That's why I start with 'open barcode' and 'click' whatever it is they need to click. I don't know if the steps I wrote are correct (probably not), and I am not sure  what the fields are called- but I recommend updating it to something along those lines.

Author: So with RFID, the scanner does the counting for us. I did add a note that users can verify the count manually if necessary and added a quick note about how to adjust the count if it's wrong for whatever reason.

==========================
Manage inventory with RFID
==========================

Managing inventory with an RFID barcode scanner is possible with the **Barcodes** app and a physical
RFID scanner.

Make sure the RFD40 scanner is paired and connected to the mobile computer or smartphone. Open an
operation (like a receipt, transfer, delivery order, or manufacturing order), or count inventory
using the RFID scanner. Scanning an existing operation barcode pulls up the relevant inventory or
manufacturing documents in Odoo on the mobile device.

Overview of the workflow
========================

> With the RFD40 scanner in hand, open the **Barcode** app on the computer or phone. Open the

Reviewer 2: ```suggestion
With the RFD40 scanner in hand, open the :menuselection:`Barcode` app on the computer or phone. Open the
```
because we're asking them to navigate there (:
